ui/src/app/views/cohort-list/component.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import {mount} from 'enzyme';
+import * as React from 'react';
+
+import {WorkspacesServiceStub, WorkspaceStubVariables} from 'testing/stubs/workspace-service-stub';
+
+import {CohortList} from 'app/views/cohort-list/component';
+
+import {registerApiClient} from 'app/services/swagger-fetch-clients';
+import {currentWorkspaceStore, urlParamsStore} from 'app/utils/navigation';
+import {CohortsApi, WorkspacesApi} from 'generated/fetch';
+import {CohortsApiStub, exampleCohortStubs} from 'testing/stubs/cohorts-api-stub';
+import {WorkspacesApiStub} from 'testing/stubs/workspaces-api-stub';
+
+import {
+  WorkspaceAccessLevel,
+} from 'generated';
+import {waitOneTickAndUpdate} from 'testing/react-test-helpers';
+
+describe('CohortList', () => {
```

Comment:
I think this spec should also have the two tests that were in the angular spec - checking that if we delete a cohort the view updates, and that if we edit a cohort the view changes

Reply:
I actually talked through with Brett on this one, and removing those was intentional. I don't think there's much value in testing it, because testing that it actually calls the `onDone` function should be done by the modal, and the logic for it here is a really simple callback that just calls `loadCohorts`, which should be tested by the basic test.